tests/integration/conftest.py

```diff
@@ -88,18 +88,25 @@ def charm_versions() -> IntegrationTestsCharms:
         ),
         zookeeper=TestCharm(
             name="zookeeper-k8s",
-            channel="3/edge",
+            channel="3/stable",
             revision=70,
```

Comment:
maybe update the revision to match the `3/stable` channel as well?

Reply:
Done in 289b0e3b8a3fd88dad6a594c21f2a862fc0b7660!